server/app/serializers/fusor/deployment_serializer.rb

```diff
@@ -21,6 +21,7 @@ class DeploymentSerializer < ActiveModel::Serializer
                :openstack_undercloud_ip_addr,
                :openstack_undercloud_user,
                :openstack_undercloud_user_password,
+               :is_autogenerate_overcloud_password,
```

Comment:
+1 to @cfchase comment. I think ```openstack_overcloud_autogenerate_password``` would line up better with the other openstack_overcloud columns.